docs/DeveloperGuide.md

```diff
@@ -336,6 +336,64 @@ results based on the specified criteria.
 
 <puml src="diagrams/FindSequenceDiagram.puml" alt="FindSequenceDiagram" />
 
+## Export feature
+
+### Implementation
+
+Step 1: The user executes `export`
+
+Step 2: The `export` command exports all data currently contained by DorManagerPro to a JSON file in the data folder.
+The name of the JSON file is the time of export.
+
+The following sequence diagram shows how a `export` command goes through the `Logic` component:
+
+### Design considerations:
+
+**Aspect: The name of the JSON file on `export`
+
+* **Alternative 1:** Use a generic format such as `SAVE_FILE_1`, `SAVE_FILE_2`
+    * Pros: Easy for the user to know at first glance which save file was created latest as well as the chronological order of their creation.
+    * Cons: Does not tell the user at what time they actually exported the save file or contents of the save file.
+
+* **Alternative 2 (current implementation):** The time of the device's system at the moment of `export`
+    * Pros: Allows for pinpoint of exact save files as no two files can be exported at the same time.
+    * Cons: Takes longer to manually understand the name of the save file when trying to locate a specific one, 
+  especially when multiple `exports` happen within a short period of time. Also has no mention of contents of JSON file.
+
+* **Alternative 3:** A brief summary of the file such as `FIRST-Alex_Jones LENGTH-20`
+    * Pros: Easy for the user to know at first glance which save file contains what.
+    * Cons: Does not tell the user at what time they actually exported the save file. Also has no indication of the 
+  chronology of creation of the files, and hence no information on which file was the last to be created. Additionally,
+  it would become complicated when exporting files with similar data.
+
+## Import feature
+
+### Implementation
+
+The `import` command implements `Undoable`. The `import` command deletes the contacts whose `GradYear` field is earlier
```

Comment:
Should this also mention that import inherits from FileAccessCommand? Also, should "deletes the contacts whose..." be updated to be relevant to import? For the next few lines as well.